Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
     name: "UINotifications",
     platforms: [
-        .iOS(.v9),
+        .iOS(.v13),
```

Comment:
<!--
  1 failure:  Collection litera...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Package.swift;
  Line: 8;
-->
- :no_entry_sign: Collection literals should not have trailing commas. (`trailing_comma`)